src/components/AppMap.ts

```diff
@@ -431,6 +434,10 @@ export default class AppMap extends mixins(MixinUtil) {
     data.features.forEach((feat: any) => {
       // Create Layer
       let layer: any = L.GeoJSON.geometryToLayer(feat);
+      // Only set style for Polylines not Markers
+      if (layer.setStyle) {
+        layer.setStyle({ color: feat.style.color || this.drawLineColor });
```

Comment:
Is feat.style guaranteed to be an object? (Do we need to check if feat.style is an object?)